pkg/controller/plan/adapter/openstack/builder.go

```diff
@@ -227,15 +585,15 @@ func (r *Builder) PodEnvironment(_ ref.Ref, _ *core.Secret) (env []core.EnvVar,
 }
 
 func (r *Builder) PersistentVolumeClaimWithSourceRef(da interface{}, storageName *string, populatorName string, accessModes []core.PersistentVolumeAccessMode, volumeMode *core.PersistentVolumeMode) *core.PersistentVolumeClaim {
-	image := da.(*openstack.Image)
+	image := da.(*model.Image)
 	apiGroup := "forklift.konveyor.io"
 	virtualSize := image.VirtualSize
 	// virtual_size may not always be available
 	if virtualSize == 0 {
 		virtualSize = image.SizeBytes
 	}
 	if *volumeMode == core.PersistentVolumeFilesystem {
-		virtualSize = int64(float64(virtualSize) * 1.1)
+		virtualSize = int64(float64(virtualSize) * 2)
```

Comment:
why 2? that's the fsOverhead, no?

Reply:
This change shouldn't be here :).
I got `no space left on device` errors in my env so I tried increasing this (which indeed fixed the issue).

Reply:
interesting
@bennyz ^

Reply:
yeah we discussed this, I'm still not clear on whether it's an issue specific to the iscsi backend of the ovirt-csi storage class. Additionally, block mode should be preferred for ovirt-csi as it supports it, and the ovirt csi driver uses ext4/xfs, neither of which should require extra 100% for the fsOverhead

Reply:
I have reverted this change but we need to figure out what's the actual size we need here to make sure the populator does not fail